docs/guidelines/platforms/android/guideline_operable_android.md

```diff
@@ -146,63 +146,37 @@ Apps should not contain elements that flash more than three times in one second.
 
 Provide ways to help users navigate, find content, and determine where they are.
 
-### Bypass Blocks
-
-*This guideline covers points 2.4.1 Bypass Blocks - Level A of the WCAG standard.*
-
-:white_check_mark: **Success criteria**
+### Bypass Blocks (WCAG 2.4.1 - Level A)
 
 The app should be implemented so that it is possible to relatively easily skip the content that is repeated on the screen or the content that is irrelevant to the user.
 
-This feature is also based on a good implementation of grouping the views displayed in the app as described in [Perceivable guidelines](https://github.com/infinum/accessibility-mobile-standards/blob/master/docs/guidelines/platforms/android/guideline_percievable_android.md).
+✅ **Success criteria**
 
-For example, accessibility services users should be able to skip the whole RecyclerView list if the content is irrelevant to them without going through each item in the list.
+The first step in satisfying the criteria is having a design that breaks content into smaller pieces and provides us with "anchor points" that can be used to skip chunks of content (for example, splitting long text into paragraphs, or grouping form fields into sections with headings). After that, these anchor points need to be properly categorized (as headings, controls, etc.) in order to become visible to assistive services and used for navigation.
 
-- Headings within text
+In addition to the basic left/right swipe navigation through elements, TalkBack offers navigation by element types (headings, controls and links) and fine-grained navigation through text (by paragraphs, lines, words and characters) which is controlled with up/down swipes, as explained in the [Reading controls chapter](https://support.google.com/accessibility/android/answer/6007066?hl=en) of Google's TalkBack support page. These alternate modes of navigation are the main tool used for bypassing blocks, so it is important to make sure they all work as intended. Fortunately, most of them work well with native components, but some require additional effort.
 
-It is also possible to use _headings_ to summarize groups of text that appear on the screen. For apps with minSdk >= 28, you can set `android:accessibilityHeading` to `true` for a view to being treated as a heading.
```

Comment:
Removed implementation details, since they have been moved to the headings chapter added in #55.